packages/medusa/src/api/routes/store/carts/update-cart.ts

```diff
@@ -144,7 +144,7 @@ async function retrieveCartWithIsolatedProductModule(req, id: string) {
       },
       fields: ["id"],
       variants: {
-        fields: ["id"],
+        fields: ["id", "product_id"],
```

Comment:
question: shouldn't the __args to filter be in __args: { filters: { } } ? cc @carlos-r-l-rodrigues.

Reply:
when the property is a primary key (`id` in this case) by default it is used as a filter.

Reply:
Also the `product_id` in this case has to be manually added because it is an internal relation of the Product module.
If it was a relation represented on the joiner configs, it would be auto infered.